lib/scripts/users_list.rb

```diff
@@ -26,4 +29,16 @@ def select_all_users
     User.all.select(:id, :formal_name, :username, :email,
                     :current_sign_in_at, :last_sign_in_at)
   end
+
+  def solr_query_by_depositor(depositor)
```

Comment:
I like these tiny methods. They make it much easier to understand what's going on here.